tests/python/rest_api/test_projects.py

```diff
@@ -708,20 +800,31 @@ def test_exported_project_dataset_structure(
             ],
         }
 
-        response = self._export_annotations(username, pid, format=anno_format)
-        assert response.data
-        with zipfile.ZipFile(BytesIO(response.data)) as zip_file:
+        dataset = self._test_export_annotations(
+            username,
+            pid,
+            api_version=api_version,
+            format=anno_format,
+            local_download=(not (project["target_storage"] or {}).get("cloud_storage_id")),
```

Comment:
It feels like local downloading should be a test parameter in a separate test and just set to True in all these tests.

Reply:
You are right, fixed